content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,375 @@
+signal below the threshold, the kubelet begins to evict end-user pods. 
+
+The kubelet uses the following parameters to determine pod eviction order:
+
+1. Pod resource usage relative to defined requests
+1. [Pod priority](/docs/concepts/configuration/pod-priority-preemption/)
+1. Resource consumption relative to scheduling requests
+1. Pod [QoS class](/docs/tasks/configure-pod-container/quality-service-pod/)
+
+As a result, kubelet ranks and evicts pods in the following order:
+
+1. `BestEffort` or `Burstable` pods where the usage exceeds requests. These pods
+   are evicted based on their Priority and then by how much their usage level
+   exceeds the request.
+1. `Guaranteed` pods and `Burstable` pods where the usage is less than requests
+   are evicted last, based on their Priority.
+
+`Guaranteed` pods are guaranteed only when requests and limits are specified for
+all the containers and they are equal. These pods will never be evicted because
+of another pod's resource consumption. If a system daemon (such as `kubelet`,
+`docker`, and `journald`) is consuming more resources than were reserved via 
+`system-reserved` or `kube-reserved` allocations, and the node only has
+`Guaranteed` or `Burstable` pods using less resources than requests left on it,
+then the kubelet must choose to evict one of these pods to preserve node stability
+and to limit the impact of resource starvation on other Pods. In this case, it
+will choose to evict pods of lowest Priority first.
+
+When the kubelet evicts pods to reclaim disk resources, it sorts pods by QoS and
+terminates pods using the largest amount of disk. When the kubelet responds to
+`inode` starvation, it uses the QoS to make eviction decisions. The kubelet sorts
+pods differently based on whether the node has a dedicated `imagefs` filesystem:
+
+### With `imagefs`
+
+If `nodefs` is triggering evictions, the kubelet sorts pods based on `nodefs`
+usage (`local volumes + logs of all containers`).
+
+If `imagefs` is triggering evictions, the kubelet sorts pods based on the
+writable layer usage of all containers.
+
+### Without `imagefs`
+
+If `nodefs` is triggering evictions, the kubelet sorts pods based on their total
+disk usage (`local volumes + logs & writable layer of all containers`)
+
+## Minimum eviction reclaim
+
+In some cases, pod eviction only reclaims a small amount of the starved resource.
+This can lead to the kubelet repeatedly hitting the configured eviction thresholds
+and triggering multiple evictions. 
+
+You can use the `eviction-minimum-reclaim` flag to configure a minimum reclaim
+amount for each resource. When the kubelet notices that a resource is starved,
+it continues to reclaim that resource until it reclaims the quantity you specify. 
+
+For example, the following configuration sets minimum reclaim amounts: 
+
+```
+--eviction-hard=memory.available<500Mi,nodefs.available<1Gi,imagefs.available<100Gi
```

Comment:
we likely want to point to the kubelet config variant of this config rather than the flags.

Reply:
Do you mean the YAML config? Where would I get my hands on that? ✋🏽

Reply:
yes, as defined here https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/

for example:

```
apiVersion: kubelet.config.k8s.io/v1beta1
kind: KubeletConfiguration
evictionHard:
    memory.available:  "200Mi"
```

full list of options are defined here: https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/